src/ui.cpp

```diff
@@ -598,15 +598,17 @@ void uilist::inputfilter()
     filter_popup.reset();
 }
 
-static ImVec2 calc_size( const std::string_view line )
+static ImVec2 calc_size( const std::string_view line, float wrap_width = -1.0f )
 {
-    return ImGui::CalcTextSize( remove_color_tags( line ).c_str() );
+    return ImGui::CalcTextSize( remove_color_tags( line ).c_str() , nullptr, false, wrap_width );
 }
 
 void uilist::calc_data()
 {
     ImGuiStyle s = ImGui::GetStyle();
+    const ImGuiStyle &style = s;
 
+    // TODO: why is this here
```

Comment:
Translation: i find it weird that hotkeys get reassigned each time the window is resized. It feels like this logic should happen elsewhere, but I haven't thought about this enough to say anything more constructive.

Reply:
Heh, now that you mention it that is pretty dumb. We can move that somewhere more appropriate, I am sure.